extension/app/background.ts

```diff
@@ -1,3 +1,114 @@
+import { randomString } from "./src/lib/utils";
+import {
+  AUTH0_AUDIENCE,
+  AUTH0_CLIENT_DOMAIN,
+  AUTH0_CLIENT_ID,
+  AuthBackroundMessage,
+  Auth0AuthorizeResponse,
+  AuthBackgroundResponse,
+} from "./src/lib/auth";
```

Comment:
Nope but not working as is we must be missing some config, will do in the PR that adds some eslint rules so that this one is focused on the oauth flow

Reply:
Likely challenging in the extension given the specific build process. But hopefully can work.